frontend/src/pages/surveys/components/step/render-steps.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  type RenderSteps,
+  type StepsType,
+  type UseSurveyState,
+} from '#packages/survey/survey.js';
+import { useStateSurvey } from '#packages/survey/survey.js';
+import { STEPS_CONFIGURATION } from '#pages/surveys/libs/constants/constants.js';
+
+import { Step } from './step.js';
+
+type Properties = {
+  currentStep: StepsType;
+  onNextStep: () => void;
+  onPreviousStep: () => void;
+  onSubmit: () => void;
+};
+
+const RenderSteps: React.FC<Properties> = ({
+  currentStep,
+  onNextStep,
+  onPreviousStep,
+  onSubmit,
+}): React.ReactNode => {
```

Comment:
as I remember we don't use this approach across the app
```suggestion
}) => {
```